enterprise/api/v1/serializers.py

```diff
@@ -638,20 +638,47 @@ def create(self, validated_data):
         cohort = validated_data.get('cohort')
         email_students = validated_data.get('email_students')
         is_active = validated_data.get('is_active')
+        LOGGER.info(
+            "[Enrollment-API] Received a call with the following parameters. lms_user: [{lms_user}], "
```

Comment:
since we are passing ids in `lms_user` and `tpa_user`, can we add "id" next to them as well in the logs? so `lms_user` becomes `lms_user_id`

Reply:
It will be enterprisecutomeruser id. lms_user or tpa_user will notify from where we did get the EnterpriseCustomerUser id.